### ISSUE TITLE
Stack not able to Compile

### ISSUE THREAD

**Issue by DharanBro** [OfficeDev/office-ui-fabric-react#8025]
### Environment Information 
 
- __Package version(s)__: 6.140.0

```
import { Component } from 'react'
import * as React from 'react';
import { mergeStyleSets, DefaultPalette } from 'office-ui-fabric-react/lib/Styling';
import { Stack } from 'office-ui-fabric-react/lib/Stack';

export default class Home extends Component {

    public render(): JSX.Element {
        const styles = mergeStyleSets({
            root: {
                background: DefaultPalette.themeTertiary
            },

            item: {
                height: 50,
                display: 'flex',
                alignItems: 'center',
                justifyContent: 'center',
                color: DefaultPalette.white,
                background: DefaultPalette.themePrimary
            }
        });
        return (
            <Stack horizontal padding={10} className={styles.root}>
                <Stack.Item grow={3} className={styles.item}>
                    <NavComponent />
                </Stack.Item>
                <Stack.Item grow={2} className={styles.item}>
                    Grow is 2
                </Stack.Item>
                <Stack.Item grow className={styles.item}>
                    Grow is 1
                </Stack.Item>
            </Stack>
        )
    }
}

``` 
```
Failed to compile.

/Volumes/My Drive/Workspace/VBI/APSTool/aps-client/node_modules/office-ui-fabric-react/lib/components/Stack/Stack.types.d.ts
(27,5): 'as' is referenced directly or indirectly in its own type annotation.
```
#### Priorities and help requested:
 
Requested priority: Blocking

**Comment by aneeshack4** [comment]
@DharanBro Can you provide some more context into what you were doing when you got this compile error? Did you make sure to "npm install" after freshly syncing ("git pull upstream master" and make sure your remotes are set correctly so that you're pulling from the right source.
![image](https://user-images.githubusercontent.com/4161791/53040987-a6ee6e00-3437-11e9-85af-2449fbc0301a.png)

**Comment by JasonGore** [comment]
This should have been fixed in [v6.141.0](https://github.com/OfficeDev/office-ui-fabric-react/releases/tag/office-ui-fabric-react_v6.141.0) due to the merge of #8017. If you are still seeing issues with v6.141.0 please reply and let us know! Thanks.
